cmd/datasource-syncer/README.md

```diff
@@ -44,10 +44,15 @@ cat datasource-syncer.yaml \
 
 To query across multiple projects, you must [create a metrics scope](https://cloud.google.com/stackdriver/docs/managed-prometheus/query#scoping-intro) and authorize the local project's default compute service account to have monitoring.read access to the scoping project. If your local project is your scoping project, then this permission is granted by default and cross-project querying should work with no further configuration.
 
+### Workload Identity Setup
+If you're using WLI you need to grant the service account these two permissions:  `Monitoring Viewer` and `Service
+Account Token Creator`.
+
+
 ### Development
 ```bash
 go run main.go \
-  --credentials-file=$CREDENTIALS \
+  --google-application-credentials=$GOOGLE_APPLICATION_CREDENTIALS \
```

Comment:
Following the example in our docs (https://cloud.google.com/stackdriver/docs/managed-prometheus/query#explicit-credentials) I think this line should be
--query.credentials-file=$GOOGLE_APPLICATION_CREDENTIALS